### PR TITLE
Remove extraneous parens around return arguments

### PR DESCRIPTION
**Pull request by jasone**
Pull request is mainly for CI coverage.  More style update diffs still to come.

### CODE REVIEW

**Comment by jasone** [comment]
Ready for review.

File: include/jemalloc/internal/atomic_inlines.h [hunk 1]
```diff
@@ -422,16 +422,16 @@ atomic_sub_p(void **p, void *x) {
 #if (LG_SIZEOF_PTR == 3)
 	return ((void *)atomic_add_u64((uint64_t *)p, (uint64_t)-((int64_t)x)));
```
Comment (interwq) [inline include/jemalloc/internal/atomic_inlines.h]: remove this one as well?

File: include/jemalloc/internal/rb.h [hunk 2]
```diff
@@ -974,7 +974,7 @@ a_prefix##reverse_iter(a_rbt_type *rbtree, a_type *start,		\
 	ret = a_prefix##reverse_iter_recurse(rbtree, rbtree->rbt_root,	\
 	  cb, arg);							\
     }									\
-    return (ret);							\
+    return ret;								\
```
Comment (interwq) [inline include/jemalloc/internal/rb.h]: tab seems off?

Comment (jasone) [inline include/jemalloc/internal/rb.h]: This has the same cause as the one you pointed out in #573.

File: include/jemalloc/internal/atomic_inlines.h (hunk 1 above)
Comment (jasone) [inline include/jemalloc/internal/atomic_inlines.h]: Good catch.

**Pull request merged**